gallery/src/components/NxList/examples/NxListBulleted.tsx

```diff
@@ -0,0 +1,55 @@
+/*
+ * Copyright (c) 2019-present Sonatype, Inc.
+ * This program and the accompanying materials are made available under
+ * the terms of the Eclipse Public License 2.0 which accompanies this
+ * distribution and is available at https://www.eclipse.org/legal/epl-2.0/.
+ */
+import React from 'react';
+import { NxH3, NxList } from '@sonatype/react-shared-components';
+
+function NxListBulleted() {
+  return (
+    <NxList bulleted>
+      <NxH3>Bulleted lists with a heading</NxH3>
```

Comment:
It isn't valid HTML to put the header within the list.  It should be outside of it and IMO doesn't need to be included in these examples at all.

Reply:
> It isn't valid HTML to put the header within the list. It should be outside of it and IMO doesn't need to be included in these examples at all.

All headers that were previously within `NxList` have been removed, both from the gallery as well as library implementation.
7d05c8feb49640bac834968a1bfd984deb457640